UniWatch/Views/Class/TeacherIndex.cshtml

```diff
@@ -0,0 +1,59 @@
+﻿@model IEnumerable<UniWatch.Models.Class>
+
+@{
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+    <h3>Courses Taught</h3>
+
+    <p>
+        @Html.ActionLink("New Class", "Create", null, new { @class = "btn btn-success" })
+    </p>
+
+@if (!Model.Any())
+{
+    @:It's empty in here. Create a new class.
+}
+else
+{
+    <div class="table-responsive">
+        <table class="table table-bordered table-striped">
+            <tr>
+                <th>
+                    @Html.DisplayNameFor(model => model.Name)
+                </th>
+                <th>
+                    @Html.DisplayNameFor(model => model.Number)
+                </th>
+                <th>
+                    @Html.DisplayNameFor(model => model.Section)
+                </th>
+                <th>
+                    @Html.DisplayNameFor(model => model.Year)
+                </th>
+                <th></th>
+            </tr>
+            @foreach (var item in Model)
+            {
+                <tr>
+                    <td>
+                        @Html.ActionLink(item.Name, "Index", "Lecture", new {classId = item.Id}, null)
+                    </td>
+                    <td>
+                        @Html.DisplayFor(modelItem => item.Number)
+                    </td>
+                    <td>
+                        @Html.DisplayFor(modelItem => item.Section)
+                    </td>
+                    <td>
+                        @Html.DisplayFor(modelItem => item.Year)
+                    </td>
+                    <td>
+                        @Html.ActionLink("Delete", "Delete", new {classId = item.Id})
+                    </td>
+                </tr>
+            }
+        </table>
+    </div>
+}
+
```

Comment:
I have look everything patrick. Thanks for fixing some stuff I have overlook. I have approved all changes in here.